packages/react/examples/drag-and-drop-file-uploader/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "carbon-components": "latest",
     "carbon-components-react": "latest",
+    "carbon-icons": "latest",
```

Comment:
Just wanted to see if this is intentional?

Reply:
yes, on `master` branch the example will not load without this dependency https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/drag-and-drop-file-uploader

Reply:
Got it - Thanks @emyarod for clarifying!